src/components/PathwayCourse/index.js

```diff
@@ -0,0 +1,227 @@
+import React, { useEffect, useState } from "react";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import CheckIcon from "@mui/icons-material/Check";
+import useStyles from "./styles";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import { PATHS, interpolatePath } from "../../constant";
+import { METHODS } from "../../services/api";
+import { useParams } from "react-router-dom";
+import { breakpoints } from "../../theme/constant";
+import { useSelector, useDispatch } from "react-redux";
+import { actions as pathwayActions } from "../PathwayCourse/redux/action";
+import {
+  Container,
+  Box,
+  Grid,
+  Card,
+  CardContent,
+  Typography,
+  CardMedia,
+  Toolbar,
+} from "@mui/material";
+
+const pathways = {
+  1: {
+    pathway: "Python",
+    description:
+      "Learn the basics and become comfortable in one of the most popular programming languages Python.",
+    outcomes: [
+      "Get equipped to build small projects like calculator or to-do list",
+      "Get the base knowledge to apply to advanced bootcamps such as Navgurukul or Zoho Schools",
+    ],
+  },
+
+  2: {
+    pathway: "Javascript",
+    description:
+      "Learn the nuances and basics of the technology that powers the web. Start with learning what is Javascript and eventually build your own website.",
+    outcomes: [
+      "Build your first web page and power it with the interactive language of Javascript",
+      "Build your basics of HTML, CSS and Javascript to prepare for advanced web development courses",
+    ],
+  },
+  3: {
+    pathway: "Typing Guru",
+    description:
+      "The typing track allows you to practice keyboard typing in a adaptive manner. You require a keyboard if on Android or use your laptop keyboard.",
+    outcomes: [
+      "Reach a typing speed of up to 30 to 40 words per minute",
+      "Be able to type long text with minimal inaccuracies",
+    ],
+  },
+  5: {
+    pathway: "Spoken English & Grammar",
+    description:
+      "English is a great tool needed to navigate the tech world and also in an International setting. Whether you are a total beginner or already know some English, prepare for the challenge with our Spoken English classes and online courses.",
+    outcomes: [
+      "Start speaking English without fear in about 6 months",
+      "Be able to read, write, listen and speak English with fluency",
+      "Be able to give oral presentations, talk to friends and prospective colleagues",
+    ],
+  },
+};
+
+const images = [
+  "course1",
+  "course2",
+  "course3",
+  "course1",
+  "course2",
+  "course3",
+  "course1",
+  "course2",
+  "course3",
+  "course1",
+];
```

Comment:
If you're going to cycle through integers, you can use the remainder operator (`%`).  This below suggestion should be improved by using appropriate constants for `10` and cycle length of `3`.
```suggestion
const images = Array.from(new Array(10).keys()).map((i) => "course" + (i % 3 + 1));
```

Reply:
Actually, this is temporary. Once we'll get images from the backend we'll remove them. Anyway, thanks this is really a good idea.